src/services/expenses.services.js

```diff
@@ -0,0 +1,47 @@
+const { getExpensesFilterQuery } = require('../utils/expenses');
+
+const { Expense } = require('../models/Expense.model');
+
+const getExpenses = async (categories, userId, from, to) => {
+  const where = getExpensesFilterQuery(categories, userId, from, to);
+
+  const expenses = await Expense.findAll({ where });
+
+  return expenses;
+};
+
+const getExpenseById = async (id) => {
+  const expense = await Expense.findByPk(id);
+
+  return expense;
+};
+
+const createExpense = async (expense) => {
+  const newExpense = await Expense.create(expense);
```

Comment:
Consider adding error handling for the `create` operation to manage potential database errors, such as validation errors or connection issues.